assemble/src/main/assemblies/component.xml

```diff
@@ -47,7 +47,7 @@
         <include>commons-cli:commons-cli</include>
         <include>commons-codec:commons-codec</include>
         <include>commons-collections:commons-collections</include>
-        <include>commons-configuration:commons-configuration</include>
+        <include>org.apache.commons:commons-collections4</include>
```

Comment:
You've incorrectly replaced the wrong artifact. This should replace commons-collections, not commons-configuration.

Reply:
I did thank you!